airflow/contrib/hooks/salesforce_hook.py

```diff
@@ -18,127 +18,108 @@
 # under the License.
 #
 """
-This module contains a Salesforce Hook
-which allows you to connect to your Salesforce instance,
-retrieve data from it, and write that data to a file
-for other uses.
+This module contains a Salesforce Hook which allows you to connect to your Salesforce instance,
+retrieve data from it, and write that data to a file for other uses.
 
-NOTE:   this hook also relies on the simple_salesforce package:
-        https://github.com/simple-salesforce/simple-salesforce
+NOTE: this hook also relies on the simple_salesforce package:
```

Comment:
```suggestion
.. note:: this hook also relies on the simple_salesforce package:
```
Reference: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-note